src/main/kotlin/no/nav/familie/ba/sak/kjerne/brev/mottaker/BrevmottakerService.kt

```diff
@@ -127,3 +138,10 @@ class BrevmottakerService(
         landkode = brevmottaker.landkode,
     )
 }
+
+private fun List<Brevmottaker>.zeroSingleOrThrow(exception: List<Brevmottaker>.() -> Exception): Brevmottaker? =
+    if (size in 0..1) {
+        singleOrNull()
+    } else {
+        throw exception()
+    }
```

Comment:
```suggestion
private inline fun <T> List<T>.zeroSingleOrThrow(exception: () -> Exception): T? {
    if (size <= 1) {
        return singleOrNull()
    } else {
        throw exception()
    }
}
```

Kunne også ha gjort noe sånt siden `zeroSingleOrThrow` kan passe med alle forskjellige lister.